src/main/java/io/stargate/sgv2/jsonapi/exception/JsonApiException.java

```diff
@@ -106,4 +135,134 @@ public ErrorCode getErrorCode() {
   public Response.Status getHttpStatus() {
     return httpStatus;
   }
+
+  private String getErrorFamily() {
+    // some error codes should be classified as SERVER errors but do not have any pattern
+    Set<ErrorCode> serverFamily =
```

Comment:
I think this is not a good way to do this -- instead of creating a `Set` here, should include this information directly in `ErrorCode`.

Reply:
we took this approach to reduce the amount of existing code we had to change, and the amount of repitition. All of the existing ErrorCode enum is going to be removed , and there is 90 odd definitions there.  So rather than change all fo them we took this approach for the lowest number of modified lines

Reply:
we should move this set to be static final though so it is not created each time the function runs

Reply:
Ok. Fair enough.